### PR TITLE
HDDS-3389. Add response to SetVolumePropertyResponse proto

### PR DESCRIPTION
**Pull request by smengcl**
## What changes were proposed in this pull request?

https://github.com/apache/hadoop-ozone/pull/806#discussion_r408279098

1. Add `optional bool response = 1;` in the message.
2. Handle the response on the client. e.g. in setOwner, if the response is false, we can print message: The specified user is already the owner of the volume.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-3389

## How was this patch tested?

```bash
bash-4.2$ ozone sh volume create vol1
bash-4.2$ vi /etc/hadoop/ozone-shell-log4j.properties
# Append: log4j.logger.org.apache.hadoop.ozone.shell.Handler=DEBUG
bash-4.2$ ozone sh volume update vol1 --user=user1
{
  "metadata" : { },
  "name" : "vol1",
  "admin" : "hadoop",
  "owner" : "user1",
  "creationTime" : "2020-05-05T12:05:10.885Z",
  "acls" : [ {
    "type" : "USER",
    "name" : "hadoop",
    "aclScope" : "ACCESS",
    "aclList" : [ "ALL" ]
  }, {
    "type" : "GROUP",
    "name" : "users",
    "aclScope" : "ACCESS",
    "aclList" : [ "ALL" ]
  } ],
  "quota" : 1152921504606846976
}
bash-4.2$ ozone sh volume update vol1 --user=user1
Volume 'vol1' owner is already 'user1'. Unchanged.
{
  "metadata" : { },
  "name" : "vol1",
  "admin" : "hadoop",
  "owner" : "user1",
  "creationTime" : "2020-05-05T12:05:10.885Z",
  "acls" : [ {
    "type" : "USER",
    "name" : "hadoop",
    "aclScope" : "ACCESS",
    "aclList" : [ "ALL" ]
  }, {
    "type" : "GROUP",
    "name" : "users",
    "aclScope" : "ACCESS",
    "aclList" : [ "ALL" ]
  } ],
  "quota" : 1152921504606846976
}
```

### CODE REVIEW

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java [hunk 1]
```diff
@@ -1690,14 +1690,10 @@ public boolean getAclsEnabled() {
...
-   * Changes the owner of a volume.
-   *
-   * @param volume - Name of the volume.
-   * @param owner - Name of the owner.
-   * @throws IOException
+   * {@inheritDoc}
    */
   @Override
-  public void setOwner(String volume, String owner) throws IOException {
+  public boolean setOwner(String volume, String owner) throws IOException {
```
Comment (xiaoyuyao) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java:1696]: Just curious if this ever return false?

Comment (smengcl) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java:1696]: > Just curious if this ever return false?

Yes. It returns false when the given user is already the owner of the volume: https://github.com/apache/hadoop-ozone/pull/876/commits/4613bd0724ee0be97651aaa7364f3921436b9183#diff-990a43aeb92fe45392bdab4484ec488bR150

Comment (smengcl) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java:1696]: Correction: Here is just to follow the change in `OzoneManagerServerProtocol`.

**Comment by xiaoyuyao** [comment]
LGTM, +1.

**Comment by smengcl** [comment]
Thanks @xiaoyuyao  for reviewing and committing.

**Pull request merged**